framework/src/interfaces/TaggingInterface.C

```diff
@@ -359,21 +381,73 @@ TaggingInterface::prepareMatrixTagLower(Assembly & assembly,
 void
 TaggingInterface::accumulateTaggedLocalResidual()
```

Comment:
These functions are so much longer now whereas before they were so short. And these changes to support a small subset of MOOSE simulations 😬

Reply:
If we always scale `_local_re`, code in TaggingInterface can be a bit shorter. Correspondingly, we will need remove the factor after residual evaluation for Bx vector. That might be a better solution.

Reply:
Actually, this can be used to remove bunch of code like
```
    if (eigen_problem->negativeSignEigenKernel())
    {
      ierr = VecScale(Bx, -1.);
      CHKERRQ(ierr);
    }
```
in `SlepcSupport.C`. Will push shortly.

Reply:
SlepcSupport and EigenProblem make sense for such operations. TaggingInterface imo it's going to be a "gotcha". Someone developing in the system wont expect this normalization to have happened in the taggingInterface

Reply:
This has to be done in TaggingInterface if we do not have two separate tagging vectors for A and B.